<html>

<head>
    <meta name="description" content="Urszula Czerwinska. About me, data science, big data, analytics, PhD">
</head>

<body>
    <style>
    <p> body {
        background-color: blue;
    }
    
    .highlightme {
        background-color: #ccffe5;
    }
    
    </p>
    </style>
</body>
<section>
    <p><em><b>Data, big data, omics data, noisy data, a lack of data, data overflow, open data.</b>Data is an increasingly critical part of life as a researcher — particularly in the field of computational biology. How to treat data? How to manage data? How to smartly produce data? How to simulate data? How to reproduce data? Data, data, data…</em></p>
    <p>But what is <b>Computational Biology</b>? In the past, it was seen as a support for biological research; a set of tools helping answer biological questions asked by researchers — real researchers that is. However, when you consider recent breakthroughs, dynamical development of this discipline and the impact it has, it is easy to see why those in the field believe that that computational biology is and will be a driving force of biological research. What is particularly striking is the change in computational biology from <b>descriptive to data-driven science</b>.</p>
    <div><span class="image fit"><img src="/images/old_Fotor.jpg" alt="" /></span></div>
    <p>It all started with genomics. When biologist trained in wet-lab work they could not treat their data themselves and this is how new discipline was born. Like all young disciplines, it had to negotiate its place in the research landscape. Today, computational biology is still expanding thanks to new technologies and the ongoing digital revolution.</p>
    <h3>Breakthroughs</h3>
    <p>An undeniable innovator of computational biology, <a href="https://www.iscb.org/ismb2016program/keynotes/batzoglou" target="_blank">Serafim Batzoglou</a>, in his keynote talk at ISMB 2016, presented his vision of the past and the future of this discipline. His talk started with human genome sequencing, through to data manipulation, to personal genomics and finally artificial intelligence. He stated research need to keep up with new technologies with a quote from the TV series
        <a href="http://www.imdb.com/title/tt2575988/" target="_blank"></a>Silicon Valley</a>:</p>
    <blockquote><strong>“I don’t want to live in a world where someone else is making the world a better place better than me”</strong></blockquote>
    <p>In his talk, <a href="https://www.iscb.org/ismb2016program/keynotes/batzoglou" target="_blank">Serafim Batzoglou</a> made the somewhat controversial comment that <span class="highlightme"><em>AI will replace medical doctors</span></em> in near future and he named his candidate for most important and promising technology: <b>deep learning</b>.</p>
    <blockquote><i><a href="https://en.wikipedia.org/wiki/Deep_learning" target="_blank">Deep learning</a> is a branch of machine learning based on a set of <a href="https://en.wikipedia.org/wiki/Algorithm" target="_blank">algorithms</a> that attempt to model high-level abstractions in data by using a deep graph with multiple processing layer; composed of multiple linear and <a href="https://en.wikipedia.org/wiki/Linear_transformation" target="_blank">non-linear transformations</a>.</i></blockquote>
    <div><span class="image fit"><img src="/images/keynotetalk.jpg" alt="" /></span></div>
    <p><b>Single-cell transcriptomics</b> were unofficially named most exciting technology for studying biological systems - with three out of six keynote speakers talking about their recent progress when treating single cell transcriptomics data.</p>
    <p><a href="https://www.iscb.org/ismb2016program/keynotes/batzoglou" target="_blank">Serafim Batzoglou</a> also mentioned clustering algorithm, called SimlR challenging existing methods of dimension reduction . The <a href="http://biorxiv.org/content/early/2016/05/09/052225" target="_blank">preprint</a> of the article is accessible at <a href="http://biorxiv.org/" target="_blank">http://biorxiv.org/</a>.</p>
    <div>
        <figure><span class="image fit"><img src="/images/simlR.png" alt="" /></span>
            <figcaption>Outline of SIMLR from “Visualization and analysis of single-cell RNA-seq data by kernelbased similarity learning” — Image credit: Bo Wang et al ,2016 , (CC BY-NC-ND 4.0)</figcaption>
        </figure>
    </div>
    <br>
    <p><a href="https://www.iscb.org/ismb2016program/keynotes/dudoit" target="_blank">Sandrine Dudoit</a> also referred to single-cell data; with particular attention to treating the problem of noise to signal ratio and data reproducibility. In her lab more robust tools of clustering are under development. For example: <i><a href="https://github.com/epurdom/clusterExperiment" target="_blank">clusterExperiment</a></i> decides on clusters based on many clustering methods and picks the ones that are the most stable.</p>
    <p>The keynote talk by Sarah Teichmann showcased a cutting-edge example of exploration of single-cell data. In publication she co-authors (<a href="http://www.ncbi.nlm.nih.gov/pubmed/26950746" target="_blank">“Tcell fate and clonality inference from single-cell transcriptomes”</a>), full-length paired T-cell receptor (TCR) sequences are reconstructed from single cell data.</p>
    <p>A key trend currently is to gather information at the highest level of accuracy and then group it into meaningful blocks of knowledge which best characterize the data and correspond to biological reality, or even better, discover unknown pieces of biological systems. This is how computational biology is driving biological research today.</p>
    <em><strong>And there will be more; more data, more challenges, more breathtaking discoveries!</strong></em>
    <br>
    <br>
    <div><span class="image fit"><img src="/images/coud_Fotor.jpg" alt="" /></span></div>
    <h3>Challenges</h3>
    <p>One them is <b>education</b>. <a href="https://en.wikipedia.org/wiki/Philip_Bourne" target="_blank">Phil Bourne</a>, <a href="http://www.nih.gov/news/health/dec2013/od-09.htm" target="_blank">Associate Director for Data Science of NIH</a>, in the workshop <a href="" target="_blank">Education in Bioinformatics — Exploiting Cloud and Virtual Resources for Training</a>, highlighted the importance of <b>cloud data</b> training to keep the pace of technologies developed by companies (e.g. Amazon, Google, iPlant). The new generation of computational biologists and Bioinformaticians, as well as senior researchers, require solid and up-to-date training. This is why the NIH proposes numerous programs facilitating cloud training. Unfortunately, it is not the case at all research institutes currently, but it definitely should be in the future.</p>
    <p>Another important issue raised by <a href="https://en.wikipedia.org/wiki/Philip_Bourne" target="_blank">Phil Bourne</a> and <a href="https://www.iscb.org/ismb2016program/keynotes/batzoglou" target="_blank">Serafim Batzoglou</a> was the importance of keeping data open. Phil mentioned in particular the <a href="http://www.nature.com/articles/sdata201618" target="_blank">FAIR Data Principles</a> which stand for:</p>
    <blockquote><em><strong>Findability, Accessibility, Interoperability, and Reusability</strong> - These serve to guide data producers and publishers as they navigate around obstacles, thereby helping to maximize the added-value gained by contemporary, scholarly publishing.</em></blockquote>
    <p><a href="https://www.iscb.org/ismb2016program/keynotes/batzoglou" target="_blank">Serafim Batzoglou</a> outlined the necessity of making data public in one of the slides in his keynote talk (quoted):</p>
    <blockquote>
        <h3>Biggest obstacle: wide, free data availability</h3>
        <h2>Ideal</h2>
        <ul>
            <li>Millions of publicity available genomes, medical records, phenotypes, enivornments</li>
            <li>Far reaching anti data-based discrimination laws; GINA great start</li>
        </ul>
        <h2>Not ideal</h2>
        <ul>
            <li>Data silos in hospitals, health care providers, gov’t, direct-to-direct consumer companies</li>
            <li>Specialized large-scale projects (1000 genomes but no phenotypes/medical records)</li>
        </ul>
        <h2>What about privacy?</h2>
        <ul>
            <li>No cancer patients have died yet because of data privacy breach</li>
            <li>My Facebook page reveals much more private stuff about me than my genome or medical record</li>
        </ul>
    </blockquote>
    <br>
<h2><em><span class="highlightme">Let’s make our data public — if 5% of us do, probably enough to make a big difference.
Let’s do it without waiting for legislation!</span></em></h2>
  

    <hr />
    <h3>Flaws</h3>
    <p>There are still some flaws with regards to big data — but it still represents big hope for computational biology.</p>
    <p>
        According to <a href="https://www.linkedin.com/in/urilaserson" target="_blank">Uri Laserson</a> from <a href="http://www.cloudera.com/" target="_blank">Cloudera</a> (a leader platform for data management and analysis), <a href="http://fr.slideshare.net/urilaserson/genomics-is-not-special-towards-data-intensive-biology" target="_blank">“Genomic is not special”</a>. He expressed the view that:</p>
    <blockquote><em>Computational biologists are reinventing the wheel for big data biology analysis</em>
    </blockquote>
    <p>Even though the statement is bold, there is some truth in it. It looks like industrial solutions for big data somehow outperform research solutions. This is why <b>more collaboration between research and industry is needed</b> to fill the gap. On the other hand, this collaboration cannot happen at the price of data ownership. Academy needs to find a solution to attract world-scale talents that right now goes where big data is linked tightly to big money.</p>
    <p>Another recurrent problem for many computational biologists and bioinformaticians is their place in biological research. Despite an improving amount of recognition of computational work in wet labs, there is still a number of places that lack of expertise in data analysis. Old analysis methods are reproduced and consequently spread used by others in the field — all because a number of researchers are using them.</p>
    <p>Programming or statistical input is not acknowledged enough and this is where there can some friction between experimentalists and theoreticians that can hinder progress. Strolling through alleys of the poster hall at ISMB 2016, overheard many accounts of interactions between wet and dry lab. Most of them recalled tension, disappointment or misunderstanding. This is why <b>education is really crucial</b>. No one cannot be an expert in all domains, but a comprehensible overview of each other worlds, common vocabulary and goals are essential to building a common future.</p>
    <hr />
    <p><b>Computational biology is in the heart of digital revolution.</b> Paradoxically a lack and an excess of data make this discipline extremely intriguing and challenging. With individual countries frequently making decisions on policies that have an impact on the direction and pace of the development of solutions, data’s impact on our future is undeniable. <b>But then again, what is data? A set of numbers?</b></p>
    <blockquote><h3><em>From technical point of view, each of us is a multidimensional data-set. Can we be easily reduced to a few dimensions? Can our time dimension be predicted from real time data? Can someone own us? Are we approaching era of digital slavery?</em></h3>
    </blockquote>
    <div><span class="image fit"><img src="/images/hole_Fotor.jpg" alt="" /></span></div>
    <p>You may be also interested in :</p>
    <p><a href="https://compbiofieldreports.plos.org/computational-biology-moving-into-the-future-one-click-at-a-time-83f81c0e293d#.qy67d53p8" target="_blank">Computational Biology: Moving into the Future One Click at a Time. Message from ISCB</a></p>
    <p><strong>Disclaimer: </strong>Any views expressed are those of the author, not necessarily those of PLOS.</p>
    <footer>
        <p>Exported from <a href="https://medium.com">Medium</a> on Setpember 13, 2016.</p>
        <p><a href="https://medium.com/plos-comp-biol-field-reports-2016/computational-biology-data-ismb16-1607154654ea#.ijpvjgnnw">View the original</a></p>
    </footer>
<!-- Go to www.addthis.com/dashboard to customize your tools --> <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-584ec4ce89deed84"></script> 
<!-- Go to www.addthis.com/dashboard to customize your tools --> <div class="addthis_inline_share_toolbox"></div>
 
</section>
</html>
